forge16/src/main/java/com/envyful/battle/tower/player/BattleTowerAttribute.java

```diff
@@ -353,6 +353,10 @@ public void finishAttempt() {
         this.currentFloor = 0;
     }
 
+    public boolean isAttempting() {
+        return this.attemptStart != -1 && currentFloor > 0;
```

Comment:
Why is the check for the current floor being greater than 0 important?

Reply:
I think the floor being > 0 is probably the actual correct check here, but I did both for sanity after looking at finishAttempt() & startAttempt(). Only having the attempt start had a side effect for some reason of making some players respawn at the battletower every time they logged out, maybe likely due to historical player battletower data. As to why in the DB their floor was set to 0 and the attempt start was not -1, I'm guessing was just a side effect of people disconnecting / crashing themselves in the battletower at some point.

I re-tested this after this change on my test server with an affected users data and its working as expected now